backend/authentication/password_validators.py

```diff
@@ -28,12 +29,16 @@ def get_help_text(self):
 
 
 class PasswordStrengthValidator:
-    def validate(self, password, user=None, resultLabel, result):
-        result = zxcvbn(password) resultLabel = "Score: \(result?.score ?? 0)\n"
-        return resultLabel
+    def validate(self, password, user=None):
+        score = zxcvbn(password)
+        if score < min_score:
+            raise ValidationError(
+                _("Your password needs to be strong."
```

Comment:
as it is, this will not have a space in between this period and the `"Try"` next up. add in a space here at the end.

Reply:
since you're using the same string here and below, let's save it as a var and then use it in both places. that way, you don't need to worry about changing because of this in two places, only one! whenever you see yourself using the same thing in multiple places, usually that means you should make it available to both as a function or a var.

i'd save it as something like `help_string` below line 31, so it's part of the validator. then you can access it by using `self.help_string`.